### PR TITLE
[docs] Cloud Reorganize Deploy

### PR DESCRIPTION
**Pull request by ddhodge**
Reorganize deploy and connect topics

@netlify /latest/yugabyte-cloud/

### CODE REVIEW

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Save your credentials in a safe place. If you lose these credentials you will no
 
 ## Viewing the cluster
 
-After you complete the wizard, the [**Clusters**](../../cloud-clusters/) page appears with the provisioning of your new cluster in progress. 
+After you complete the wizard, the [**Clusters**](../../cloud-clusters/) page appears with the provisioning of your new cluster in progress.
 
 ![Cluster being provisioned](/images/yb-cloud/cloud-cluster-provisioning.png)
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md:112]: line 113 suggestion: Once the cluster is ready, the [cluster overview](../../cloud-clusters/overview) is displayed OR Once the cluster is ready, the cluster [overview](../../cloud-clusters/overview) is displayed.

File: docs/content/latest/yugabyte-cloud/cloud-connect/_index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: Connect to clusters in Yugabyte Cloud
+linkTitle: Connect to clusters
+description: Connect to clusters in Yugabyte Cloud.
+headcontent: Connect to your cluster using the cloud shell, a client shell, and from applications
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/_index.md:5]: Add period

File: docs/content/latest/yugabyte-cloud/cloud-connect/connect-client-shell.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+- [ysqlsh](../../../admin/ysqlsh) — Overview of the command line interface (CLI), syntax, and commands.
+- [YSQL API](../../../api/ysql) — Reference for supported YSQL statements, data types, functions, and operators.
+- [ycqlsh](../../../admin/ycqlsh) — Overview of the command line interface (CLI), syntax, and commands.
+- [YCQL API](../../../api/ycql) — Reference for supported YCQL statements, data types, functions, and operators.
+
+## Next steps
+
+- [Add database users](../add-users/)
+- [Connect an application](../connect-application)
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/connect-client-shell.md:97]: Broken link.

File: docs/content/latest/yugabyte-cloud/cloud-connect/connect-cloud-shell.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+- [YSQL API](../../../api/ysql) — Reference for supported YSQL statements, data types, functions, and operators.
+- [YCQL API](../../../api/ycql) — Reference for supported YCQL statements, data types, functions, and operators.
+- [ysqlsh](../../../admin/ysqlsh) — Overview of the command line interface (CLI), syntax, and commands.
+- [ycqlsh](../../../admin/ycqlsh) — Overview of the command line interface (CLI), syntax, and commands.
+
+## Next steps
+
+- [Create a database](../create-databases)
+- [Add database users](../add-users/)
+- [Connect an application](../connect-application)
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/connect-cloud-shell.md:74]: Broken link.

File: docs/content/latest/yugabyte-cloud/cloud-connect/create-databases.md [hunk 5]
```diff
@@ -130,7 +129,7 @@ SELECT id, title, category, price, rating
 
 ## More to explore
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/create-databases.md:131]: Line 149 : Broken link

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md [hunk 6]
```diff
@@ -84,13 +84,13 @@ If you are creating a **Paid** cluster, set the following additional options:
...
   - Disk size/Node - enter the disk size per node in GB.
 
-The cluster costs are estimated automatically under **Cost**. **+ Usage** refers to any potential overages from exceeding the free allowances for disk storage, backup storage, and data transfer. For information on how clusters are costed, refer to [Cluster costs](../../cloud-admin/cloud-billing-costs/). 
+The cluster costs are estimated automatically under **Cost**. **+ Usage** refers to any potential overages from exceeding the free allowances for disk storage, backup storage, and data transfer. For information on how clusters are costed, refer to [Cluster costs](../../cloud-admin/cloud-billing-costs/).
 
-Paid clusters support both horizontal and vertical scaling; you can change the cluster configuration after the cluster is created using the **Edit Configuration** settings. Refer to [Configure clusters](../../cloud-clusters/configure-clusters#infrastructure). 
+Paid clusters support both horizontal and vertical scaling; you can change the cluster configuration after the cluster is created using the **Edit Configuration** settings. Refer to [Configure clusters](../../cloud-clusters/configure-clusters#infrastructure).
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md:89]: Yes. Will be fixed in another PR.

File: docs/content/latest/yugabyte-cloud/cloud-connect/_index.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+showAsideToc: true
+---
+
+Connect to clusters in Yugabyte Cloud in the following ways:
+
+- From a browser - Use cloud shell to connect to your database using any modern browser. No need to set up an IP allow list, all you need is your database password.
+- From your desktop - Install the ysqlsh or ycqlsh client shells to connect to your database from your desktop. Your computer must be added to the cluster IP allow list and an SSL connection is required.
+- Applications - Connect your applications to your cluster database.
+
+Once you have connected to the database, you can add database users so that other team members can access the database.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/_index.md:26]:
```suggestion
Once you have connected to the database, you can add database users for other team members to access the database.
```

File: docs/content/latest/yugabyte-cloud/cloud-develop/_index.md [hunk 8]
```diff
@@ -8,7 +8,7 @@ menu:
   latest:
     identifier: cloud-develop
     parent: yugabyte-cloud
-    weight: 25
+    weight: 20
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-develop/_index.md:11]: Headcontent is missing a period : Get started developing applications for Yugabyte Cloud

File: docs/content/latest/yugabyte-cloud/cloud-basics/_index.md [hunk 9]
```diff
@@ -1,59 +1,74 @@
 ---
-title: Get started with Yugabyte Cloud
-linkTitle: Get started
-description: Basics of cluster management in Yugabyte Cloud.
-headcontent: Manage clusters with Yugabyte Cloud
+title: Deploy clusters in Yugabyte Cloud
+linkTitle: Deploy clusters
+description: Deploy clusters in Yugabyte Cloud.
+headcontent: Deploy production-ready clusters
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-basics/_index.md:5]: Needs a period.

File: docs/content/latest/yugabyte-cloud/cloud-connect/connect-client-shell.md [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+
+| sslmode | MITM protection | Notes |
+|---|---|---|
+| allow | No | Effectively works as _require_ (always uses the SSL connection without verification). |
+| prefer | No | Effectively works as _require_ (always uses the SSL connection without verification). |
+| require | No | Uses the SSL connection without verification. You do not need to provide the _sslrootcert_ parameter. |
+| verify-ca | Yes | Uses the SSL connection and verifies that the server certificate is issued by a trusted certificate authority (CA). Requires the _sslrootcert_ parameter with the path to the cluster certificate. |
+| verify-full | Yes | Uses the SSL connection and verifies that the server certificate is issued by a trusted CA and that the requested server host name matches that in the certificate. Requires the _sslrootcert_ parameter with the path to the cluster certificate. |
+
+If you do not provide an `sslmode`, the connection defaults to `prefer`.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/connect-client-shell.md:70]:
```suggestion
If you don't provide an `sslmode`, the connection defaults to `prefer`.
```

File: docs/content/latest/yugabyte-cloud/cloud-connect/connect-cloud-shell.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+menu:
+  latest:
+    identifier: connect-cloud-shell
+    parent: cloud-connect
+    weight: 10
+isTocNested: true
+showAsideToc: true
+---
+
+Use any browser to connect to Yugabyte Cloud by using the cloud shell. Cloud shell does not require a CA certificate or any special network access configured.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/connect-cloud-shell.md:16]:
```suggestion
Use any browser to connect to Yugabyte Cloud by using the cloud shell. Cloud shell doesn't require a CA certificate or any special network access configured.
```

File: docs/content/latest/yugabyte-cloud/cloud-connect/create-databases-ycql.md [hunk 12]
```diff
@@ -1,16 +1,15 @@
 ---
-title: Create and explore a database using YCQL
-headerTitle: Create and explore a database using YCQL
+title: Create and explore a database
 linkTitle: Create a database
-description: Create and explore a database using YCQL.
+description: Use distributed CQL to explore core features of YugabteDB.
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/create-databases-ycql.md:4]:
```suggestion
description: Use distributed CQL to explore core features of YugabyteDB.
```

File: docs/content/latest/yugabyte-cloud/cloud-connect/connect-cloud-shell.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+You have the option of using the following command line interfaces (CLIs) in the cloud shell:
+
+- [ysqlsh](../../../admin/ysqlsh/) - YSQL shell for interacting with YugabyteDB using the [YSQL API](../../../api/ysql)
```
Comment (aishwarya24) [inline docs/content/latest/yugabyte-cloud/cloud-connect/connect-cloud-shell.md:20]: period? also on the next line.

**Review by aishwarya24 — APPROVED** [review]

**Pull request merged**